### PR TITLE
WIP: Footer

### LINKED ISSUE #30: Implement Footer
- [ ] Implement footer based on their footer experience fragment
- [ ] Navigation with content pages (collapsible on mobile)
- [ ] Newsletter input field
- [ ] Social media icons

Out of scope
- [ ] Implementation of newsletter signup logic
- [ ] Help overlay

### PR DESCRIPTION
**Pull request by jckautzmann**
Fix #30 

Test URLs:
- Before: https://main--asprey--hlxsites.hlx.page/gb/en
- After: https://footer--asprey--hlxsites.hlx.page/gb/en

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -104,6 +104,16 @@ export function getLanguage(defaultLang = 'gb/en') {
   return `${country}/${language}` || defaultLang;
 }
 
+export function getLanguageRoot(defaultRoot = '/gb/en') {
```
Comment (hannessolo) [inline scripts/scripts.js:107]: Can you use https://github.com/hlxsites/asprey/blob/69931bfeca6e86d9a31a28a4f5b161d081c775c2/scripts/scripts.js#L96 temporarily - that way I will be able to find all occurences of this to replace once I merge #66

**Review by hannessolo — COMMENTED** [review]
The borders between the sections of the footer on mobile are still missing

**Comment by jckautzmann** [comment]
> The borders between the sections of the footer on mobile are still missing

@hannessolo Good catch! I hadn't seen them as they blend into the background color.
I addressed in https://github.com/hlxsites/asprey/pull/50/commits/643b9f8bf07d86306fd7eac591325a8e7fdda5e4

File: blocks/footer-newsletter/footer-newsletter.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  letter-spacing: 0;
+  color: var(--clr-white);
+  border: 0;
+  padding: 0 1pc;
+  margin-bottom: 10px;
+  max-width: 495px;
+}
+
+.footer-newsletter input[type="email"] {
+  width: 50%;
```
Comment (herzog31) [inline blocks/footer-newsletter/footer-newsletter.css:23]: I would recommend moving `width: 50%` to `.footer-newsletter`. Make sure it's 100% on mobile, and 50% on desktop.

Then both inputs can fill the full width of the newsletter block by adding `flew-grow: 1` to each.

File: scripts/scripts.js (hunk 1 above)
Comment (hannessolo) [reply scripts/scripts.js]: Sorry, this changed again: https://github.com/hlxsites/asprey/blob/1b3b3cdc45e8301de07f3bf7dc7ed8da30552bd0/scripts/scripts.js#L28

This is the final version, I promise 😅

File: blocks/footer-newsletter/footer-newsletter.css (hunk 2 above)
Comment (jckautzmann) [reply blocks/footer-newsletter/footer-newsletter.css]: @herzog31 I re-arranged the width properties in https://github.com/hlxsites/asprey/pull/50/commits/1da279f53f28337d5a80da0e68c66605651849d4. It's different from what you suggested as it didn't quite work for me. Let me know what you think.

Comment (herzog31) [reply blocks/footer-newsletter/footer-newsletter.css]: Looks good 👍

**Review by herzog31 — APPROVED** [review]

**Pull request merged**